Assets/HoloToolkit/Sharing/Prefabs/ManualIpConfigUtility.prefab

```diff
@@ -0,0 +1,3126 @@
+%YAML 1.1
```

Comment:
BTW, your first comment is because of [this bug](https://github.com/Microsoft/HoloToolkit-Unity/issues/804) which has a PR https://github.com/Microsoft/HoloToolkit-Unity/pull/805